src/generate.py

```diff
@@ -254,7 +254,7 @@ def build_tarball(tarball_name):
     os.remove(nice_name)
 
 
-def main(lang, src_info, version, lang_config, gendate,
+def main(lang, src_info, version, lang_config, gendate, images_dump_dir,
          verbose=False, desconectado=False, process_articles=True):
```

Comment:
Podría ser mejor igualar el idioma y dejar todo en inglés
```suggestion
         verbose=False, disconnected=False, process_articles=True):
```

Reply:
El branch ya es demasiado grande como para ponerme a traducir variables/comentarios que no toco. Queda para después.